extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.css

```diff
@@ -81,7 +81,13 @@ amp-story-player a {
   margin: 0 5px !important;
   white-space: nowrap !important;
   cursor: pointer !important;
-  outline: none;
+  outline: none !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip[disabled] {
+  pointer-events: none !important;
```

Comment:
This shouldn't be needed given that you have an extra `if (not disabled)` in your JavaScript

Reply:
Had to set cursor: initial so it doesn't change the cursor to pointer here (since that's what the `pointer-events` was there for)